translator/onnx_export.py

```diff
@@ -60,14 +59,8 @@ def to_onnx(model_checkpoint='model.ckpt', onnx_filename='model.onnx'):
     print(torch_logits)
 
 def sample(onnx_model, words=['лепота', 'несть']):
-
     session = ort.InferenceSession('model.onnx')
 
-    logits = session.run(None, {
-        'inputs': inputs.numpy(),
-        'accents': accents.numpy(),
-    })
```

Comment:
BUG: Reference to `inputs` and `accents` before they were created.

Exactly the same code snippet exists after variables' initialization.